Build.PL

```diff
@@ -8,6 +8,7 @@ use 5.008_001;
 use strict;
 use warnings;
 use utf8;
+use lib '.';
```

Comment:
Build.PL is auto-generated file by [Minilla](https://github.com/tokuhirom/Minilla/) so we should not change it. If we need such change, we should fix Minilla code.